### DISCUSSION TITLE
The future of LinuxForHealth FHIR

### DISCUSSION THREAD

**Discussion by cpeter0702** [LinuxForHealth/connect#356] [Q&A]
Hi everyone,
        How are you doing? Does anyone know who currently owns the Linux for Health's FHIR? I have some questions regarding the future development of this project. I previously raised these questions through an issue ([https://github.com/LinuxForHealth/FHIR/issues/4228](https://github.com/LinuxForHealth/FHIR/issues/4228)), but I have not received any response. I'm unsure about whom to approach or how to inquire further. Does anyone know any information about it? Thanks and have a great one.

**Comment by torimcc** [comment]
@BHart670 I @lmsurpre

**Reply by BHart670** [reply]
@torimcc perhaps we need to setup time to discuss as a group?
